engine/tilesets/tileset_palettes.asm

```diff
@@ -123,12 +123,12 @@ LoadMansionPalette:
 	call FarCopyWRAM
 	ld a, BANK(wBGPals1)
 	ld de, wBGPals1 palette PAL_BG_WATER
-	ld hl, MansionPalette1 + 6 palettes
+	ld hl, MansionPalette1 palette PAL_BG_ROOF
```

Comment:
This should be "`MansionPalette1 palette 6`"; it's the `WATER` palette, not the `ROOF` palette, and just happens to be at index 6.

Reply:
My reasoning was that the roof palette is later overwritten with palette 8, so it's nice to know you're sourcing it from the location of the ROOF palette in case you want to re-order or otherwise modify it. On second thought, that's not really useful here.